src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -249,6 +253,16 @@ await _usersAndRolesDbContext.UserInvites
                             createdById: _userService.GetUserId());
                     }
 
+                    foreach (var userPublicationRole in userPublicationRoles)
+                    {
+                        await _userPublicationInviteRepository.Create(
+                            publicationId: userPublicationRole.PublicationId,
+                            email: email,
+                            publicationRole: userPublicationRole.PublicationRole,
+                            emailSent: true, // EES-3403
+                            createdById: _userService.GetUserId());
+                    }
```

Comment:
This is a bit inefficient, shouldn't we do something like `UserReleaseInviteRepository.CreateManyIfNotExists`?